doc/api/http.md

```diff
@@ -681,7 +681,9 @@ Defaults to `'utf8'`.
 The `callback` argument is optional and will be called when this chunk of data
 is flushed.
 
-Returns `request`.
+Returns `true` if the entire data was flushed successfully to the kernel
+buffer. Returns `false` if all or part of the data was queued in user memory.
+`'drain'` will be emitted when the buffer is free again.
```

Comment:
Optional micro-nit: Should `'drain'` link to the doc entry for that event?

Reply:
Firstly, the `drain` event of `request` is a little different from [`net.socket`'s `'drain'` event](https://nodejs.org/dist/latest-v9.x/docs/api/net.html#net_event_drain) because of [the code](https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js#L833). Secondly, there is no `'drain'` item in the doc. Therefore, I don't know where the `'drain'` should link to.

Reply:
Fair enough. :-D